service/src/main/java/uk/gov/hmcts/reform/fpl/model/Address.java

```diff
@@ -40,4 +48,20 @@ public Address(@JsonProperty("AddressLine1") final String addressLine1,
         this.postcode = postcode;
         this.country = country;
     }
+
+    @JsonIgnore
+    public String getAddressAsString() {
```

Comment:
why?

Reply:
As part of the template we need the full address expanded as string.